ibis/backends/dask/aggcontext.py

```diff
@@ -0,0 +1,234 @@
+import operator
+from typing import Any, Callable, Dict, Tuple, Union
+
+import dask.dataframe as dd
+from dask.dataframe.groupby import SeriesGroupBy
+
+import ibis
+import ibis.expr.datatypes as dt
+from ibis.backends.pandas.aggcontext import (
+    AggregationContext,
+    compute_window_spec,
+    construct_time_context_aware_series,
+    get_time_col,
+    window_agg_udf,
+    wrap_for_agg,
+)
+
+# TODO Consolidate this logic with the pandas aggcontext.
```

Comment:
do we have an issue to track?

Reply:
https://github.com/ibis-project/ibis/issues/5911